extensions/amp-analytics/0.1/amp-iframe-transport-message-queue.js

```diff
@@ -0,0 +1,137 @@
+/**
+ * Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {dev} from '../../../src/log';
+import {AMP_ANALYTICS_3P_MESSAGE_TYPE} from '../../../src/3p-analytics-common';
+import {SubscriptionApi} from '../../../src/iframe-helper';
+
+/** @private @const {string} */
+const TAG_ = 'amp-analytics.IframeTransportMessageQueue';
+
+/** @private @const {number} */
+const MAX_QUEUE_SIZE_ = 100;
+
+/**
+ * @visibleForTesting
+ */
+export class AmpIframeTransportMessageQueue {
```

Comment:
actually, to be consistent with other files in the folder, could you remove the leading `Amp` from its name?